src/main/java/org/fcrepo/spec/testsuite/authz/WebACModes.java

```diff
@@ -669,4 +670,150 @@ public void appendNotWriteLdpCMust(final String uri) {
+            // POST requests to a LDP-NR with acl:Append only MUST be denied
+            final Response postRequest2 = doPostUnverified(description, null, null, false);
+            postRequest2.then().statusCode(403);
+
+            // PUT requests to a LDP-NR with acl:Append only MUST be denied
+            final Response putRequest2 = doPutUnverified(description, null, null, false);
+            putRequest2.then().statusCode(403);
+
+            // DELETE requests to a LDP-NR with acl:Append only MUST be denied
+            final Response deleteRequest2 = doDeleteUnverified(description, false);
+            deleteRequest2.then().statusCode(403);
+        }
+
+    }
+
+    /**
+     * 5.7.3-B acl:Append for LDP-NR MUST PATCH test conditions.
+     *
+     * @param uri of base container of Fedora server
+     */
+    @Test(groups = { "MUST" })
+    @Parameters({ "param1" })
+    public void appendNotWriteLdpNrPatchMust(final String uri) {
+        final TestInfo info = setupTest("5.7.3-B",
+                "When a client has acl:Append but not acl:Write for an LDP-NR they MUST " +
+                        "deny all PATCH requests that delete or modify existing content.",
+                "https://fedora.info/2018/06/25/spec/#append-ldpnr", ps);
+
+        final Headers headers = new Headers(new Header("Content-type", "text/plain"));
+        final Response postLdpNr = doPost(uri, headers, "test image");
+        final String resourceUri = getLocation(postLdpNr);
+
+        createAclForResource(resourceUri, "user-read-append.ttl", this.username);
+
+        final String description = getLdpNrDescription(resourceUri);
+        final boolean patchLDPNR = isOperationAllowed(resourceUri, "PATCH");
+
+        if (description != null) {
+            // perform PATCH which also deletes.
+            final Response patchDelete = doPatchUnverified(description, new Headers(new Header("Content-type",
+                    "application/sparql-update")),
+                    "prefix dc: <http://purl.org/dc/elements/1.1/> DELETE { <> dc:title ?o1 .}" +
+                            " INSERT { <> dc:title \"I made a change\" .} WHERE { <> dc:title ?o1 .}",
+                    false);
+            // Verify failure.
+            patchDelete.then().statusCode(403);
+
+            final Response patchDeleteData = doPatchUnverified(description, new Headers(new Header("Content-type",
+                    "application/sparql-update")),
+                    "prefix dc: <http://purl.org/dc/elements/1.1/> DELETE DATA { <> dc:title \"Some title\" .}",
+                    false);
+            // Verify failure.
+            patchDeleteData.then().statusCode(403);
+        }
+
+        if (patchLDPNR) {
+            final Response headResponse = doHead(resourceUri, false);
+            final String allowPatch = getHeaders(headResponse, "Allow-Patch").findFirst().map(Header::getValue).get();
+            // How are we testing patching an LDP-NR? Xdiff?
```

Comment:
I'll add this one to the agenda for tomorrow's meething.

Reply:
according to @awoods:  "PATCH is undefined for LDP-NR".  You can remove the final if block.